CHANGELOG.md

```diff
@@ -3,6 +3,72 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.23.1
+_Dec 5, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve German locale on the Data Grid component
+- 🐞 Bugfixes
```

Comment:
Do we have any other highlights?

Reply:
Not for eXplore
For a patch I think the current one are fine :ok_hand: